pyqtgraph/graphicsItems/ViewBox/ViewBox.py

```diff
@@ -235,6 +252,26 @@ def __init__(self, parent=None, border=None, lockAspect=False, enableMouse=True,
         if name is None:
             self.updateViewLists()
 
+        self.allow_signal_relay: bool = allow_signal_relay
+
+    # TODO: make this a decorator so that sub-types
```

Comment:
We can probably just call a decorator version of this around all (user-defined) signal handlers right?